<ui version="4.0" >
 <class>MonitorDialog</class>
 <widget class="QDialog" name="MonitorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Monitor Settings</string>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Resolution decimation:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1" >
   <property name="geometry" >
    <rect>
     <x>14</x>
     <y>37</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Monitor name:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="uiChkBoxEnableRawAudio" >
   <property name="geometry" >
    <rect>
     <x>19</x>
     <y>201</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enable Raw Audio</string>
   </property>
  </widget>
  <widget class="QComboBox" name="decimationCombo" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rowsEdit" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="colsEdit" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>58</x>
     <y>160</y>
     <width>92</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Columns [1-8]:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>58</x>
     <y>120</y>
     <width>92</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Rows [1-8]:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>37</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="smoGroup" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>30</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="title" >
    <string>Spot monitor</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="boardCombo" >
    <property name="geometry" >
     <rect>
      <x>101</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="smoLeftMarginEdit" >
    <property name="geometry" >
     <rect>
      <x>101</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="smoTopMarginEdit" >
    <property name="geometry" >
     <rect>
      <x>101</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="smoSpacingEdit" >
    <property name="geometry" >
     <rect>
      <x>101</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Left margin:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Top margin:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>150</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Spacing:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Board:</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>240</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>decimationCombo</tabstop>
  <tabstop>rowsEdit</tabstop>
  <tabstop>colsEdit</tabstop>
  <tabstop>smoGroup</tabstop>
  <tabstop>boardCombo</tabstop>
  <tabstop>smoLeftMarginEdit</tabstop>
  <tabstop>smoTopMarginEdit</tabstop>
  <tabstop>smoSpacingEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MonitorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MonitorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
